src/ZoraNFTCreatorV1.sol

```diff
@@ -17,6 +17,9 @@ import {IContractMetadata} from "./interfaces/IContractMetadata.sol";
 contract ZoraNFTCreatorV1 is OwnableUpgradeable, UUPSUpgradeable, IContractMetadata, Version(8) {
     string private constant CANNOT_BE_ZERO = "Cannot be 0 address";
 
+    /// Contract names do not match
+    error UpgradeToMismatchedContractName(string expected, string actual);
```

Comment:
good q - i can see the advantage of creating the new interface, but it would be easier to put it in IContractMetadata even though it doesn't exactly belong there, but the nice thing about putting it in `IContractMetadata` is that its accessible to other contracts that inherift form `IContractMetadata`.  I'd say either way is fine

Reply:
cool will do!